mongodbatlas/fw_resource_mongodbatlas_alert_configuration.go

```diff
@@ -291,6 +291,10 @@ func (r *AlertConfigurationRS) Schema(ctx context.Context, req resource.SchemaRe
 						},
 						"interval_min": schema.Int64Attribute{
 							Optional: true,
+							Computed: true,
+							PlanModifiers: []planmodifier.Int64{
+								int64planmodifier.UseStateForUnknown(),
+							},
```

Comment:
Essentially to simplify the terraform plan output. The `interval_min` attribute will only change its value when modified by the user.
As a counter example, alert configuration has an `updated` attribute that is computed and is not set with this modifier. In this case is seems appropriate to have the message `(known after apply)` shown on the terraform plan because on every apply it will be modified.